test/fixtures/doc-yaml/user-guide/liveness/exec-liveness.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     - /bin/sh
     - -c
     - echo ok > /tmp/health; sleep 10; rm -rf /tmp/health; sleep 600
-    image: busybox
+    image: {{.BusyBoxImage}}
```

Comment:
should we rename all of these yaml files to `.yaml.in` since now they're templates?

Reply:
Done.